modules/Media.js

```diff
@@ -10,7 +10,14 @@ class Media extends React.Component {
       PropTypes.string,
       PropTypes.object,
       PropTypes.arrayOf(PropTypes.object.isRequired)
-    ]).isRequired,
+    ]),
+    queries: PropTypes.shape({
+      [PropTypes.string]: PropTypes.oneOfType([
```

Comment:
This isn't valid. You can't use `PropTypes.string` as the key of an object because it will just be converted to a string. Instead, let's use:

```js
queries: PropTypes.objectOf(
  PropTypes.oneOfType(...)
)
```

To reduce on code duplication, let's also create a type for the `oneOfType` and call it `queryType`.

```js
const queryType = PropTypes.oneOfType(...)
```

Then we can reuse that in both `propTypes.query` and `propTypes.queries`.